workers/data_refinery_workers/processors/test_smasher.py

```diff
@@ -162,6 +164,17 @@ def test_smasher(self):
             final_frame.std(axis=1)
             final_frame.median(axis=1)
 
+            final_context['output_file']
```

Comment:
This line looks like it's not doing much. Also I'm now curious about the lines above it where we're calling methods on `final_frame`. Are those doing anything? It doesn't look like we're testing any of the outputs or using `final_frame` in the rest of the test.